app/templates/components/events/event-import-section.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="ui stackable centered grid">
+  <div class="sixteen wide column">
+    <h3 class="ui header">{{t 'Import Event'}}</h3>
```

Comment:
Using `ui dividing header` would give you a header with a line. So you'll not need `<div class="ui divider"></div>` anymore.